vault/config_test.go

```diff
@@ -28,7 +28,7 @@ region=us-east-1
 [profile withmfa]
 source_profile=user2
 role_arn=arn:aws:iam::4451234513441615400570:role/aws_admin
-mfa_serial=arn:aws:iam::1234513441:mfa/blah
+mfa_Serial=arn:aws:iam::1234513441:mfa/blah
```

Comment:
This turns into an incidental test, could you create a more explicit `TestCaseSensitivity` test or similar?

Reply:
Sure. Will do.

Reply:
I've added an explicit test and additional check for key case insensitivity (previously just section) also.
Let me know if ok and if you want me to rebase.